### PR TITLE
Updated 5x50 cases 

### LINKED ISSUE #19198: 5x50 hull multiplication
Any ammo casing using the old system is spawning in full stacks instead of individually when a cartridge is fired.

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**
Fixes #19198

### CODE REVIEW

File: data/json/items/ammo.json [hunk 1]
```diff
@@ -720,19 +720,15 @@
         "ammo_type" : "NULL",
         "flags" : ["UNRECOVERABLE"]
     },
-    {   "type" : "AMMO",
+    {   "type" : "GENERIC",
         "id" : "5x50_hull",
-        "category" : "spare_parts",
-        "price" : 80000, "//" : "Should be $4 per.",
+        "copy-from" : "casing",
```
Comment (mugling) [inline data/json/items/ammo.json]: Won't this create brass casing when this should really be more similar to a shotgun hull?

Comment (Rivet-the-Zombie) [inline data/json/items/ammo.json]: Yeah they're supposed to be plastic.

Comment (Reclusive-reptile) [inline data/json/items/ammo.json]: Material is defined as plastic, so all is well.

Comment (mugling) [inline data/json/items/ammo.json]: Good point

**Pull request merged**